examples/active.ini

```diff
@@ -1,9 +1,11 @@
 [acq_params]
+adc_offset = 33
```

Comment:
These are parameters that have been added in practice but doesn't look like we updated the example to match so I add these here